src/main/java/ladder/view/MaxLadderHeightInputView.java

```diff
@@ -0,0 +1,35 @@
+package ladder.view;
+
+import ladder.util.InputViewUtil;
+
+import java.util.Optional;
+
+public class MaxLadderHeightInputView {
+    public static final int ZERO = 0;
+
+    private final int value;
+
+    public MaxLadderHeightInputView() {
+        this(getUserInput());
+    }
+
+    public MaxLadderHeightInputView(int value) {
+        validate(value);
+        this.value = value;
+    }
+
+    private static int getUserInput() {
+        System.out.println("최대 사다리 높이는 몇 개인가요?");
+        return InputViewUtil.getUserInputParseInt();
+    }
+
+    private void validate(int value) {
+        Optional.ofNullable(value)
+                .filter(elem -> elem > ZERO)
+                .orElseThrow(IllegalArgumentException::new);
```

Comment:
primitive type 에 stream 과 lambda 를 적용할 때 이와 같이 작성하는 것이 괜찮은가요?
혹시 더 좋은 방법이 있거나 해당 방법이 옳지 않다면 말씀 부탁드립니다~

Reply:
value가 int type인데 ofNullable로 체크 안해도 될 것 같습니다.

아래 제안 드린 것 처럼 value의 값만 체크해서 유효 검증을해보면 어떨까요?🤔
```suggestion
        if(value <= ZERO) {
            throw new IllegalArgumentException();
        }
```

Reply:
primitive type 은 굳이 stream 과 lambda 를 사용하지 않는 편이 좋겠네요:) 피드백 감사합니다~